lib/src/driver.dart

```diff
@@ -388,6 +406,19 @@ class Driver {
     return null;
   }
 
+  Map<String, List<fileSystem.Folder>> _getPackageMap(Packages packages) {
+    if (packages == null) {
+      return null;
+    }
+
+    Map<String, List<fileSystem.Folder>> folderMap =
+        new Map<String, List<fileSystem.Folder>>();
+    packages.asMap().forEach((String path, Uri uri) {
+      folderMap[path] = [PhysicalResourceProvider.INSTANCE.getFolder(uri.path)];
```

Comment:
This seems incorrect for Windows.  `uri.path` will always be a posix-style path (using forward slashes), whereas PhysicalResourceProvider.INSTANCE.getFolder() expects a native OS path (which on Windows will use backslashes).  I think we need to do `path.toFilePath(Platform.isWindows)`.